### ISSUE TITLE
[RFE] Create a directory with the test in the current working directory

### ISSUE THREAD

**Issue by zdohnal** [teemtee/tmt#1979]
```tmt test create <dir>``` creates a dir in test root directory, regardless of the current working directory - it would be great if the directory was created in the current working directory, see:

```
$ pwd
/home/user/tests/Regressions
$ tmt test create --template beakerlib new-test
```

Current results:
```
Directory '/home/user/tests/new-test' created.
Test metadata '/home/user/tests/new-test/main.fmf' created.
Test script '/home/user/tests/new-test/test.sh' created.
```

Expected results:
```
Directory '/home/user/tests/Regressions/new-test' created.
Test metadata '/home/user/tests/Regressions/new-test/main.fmf' created.
Test script '/home/user/tests/Regressions/new-test/test.sh' created.
```

**Comment by psss** [comment]
Sounds reasonable and should be easy to implement.

**Comment by lukaszachy** [comment]
I'm not sure. The help states `tmt test create [OPTIONS] NAME`, see the `NAME` not `<dir>`.

How should user let tmt know to create directory in the current working directory instead of the name? New switch?
IF we change the default we have usage breakage.

**Comment by ivet1987** [comment]
As a quite new person to tmt I was also confused by this behaviour. But when I see help, there is "  Specify directory name or use '.' to create tests under the current working  directory." I would also appreciate the choice of directory, or to sync these two options  $  tmt test create --template beakerlib new-test .  /With "." at the end

**Comment by lukaszachy** [comment]
Yeah, the help message doesn't help at all. IMO (for the current behavior) it should state among lines of `Specify the desired test name or use '.' to create the test inside the current working directory.

**Comment by ivet1987** [comment]
@lukaszachy @psss So what do you think? Is it worth it to implement this issue? I don't know what are all the dependencies and what's the best solution? If a new switch or just create new test in currect directory as default...

**Comment by psss** [comment]
Hm, yes, the help message is quite ambiguous, which gives us some freedom with the implementation and breaking the compatibility :) But seriously, I don't expect there would be hundreds of scripts used for mass test creation which would be broken if we slightly modify the behaviour here.

What about making it close to how paths are handled in the shell? Let's say my current working directory is:

    /home/psss/git/tmt/tests/prepare

There would be following options how to create a new test:

    tmt test create install                 ---> /tests/prepare/install
    tmt test create ./install               ---> /tests/prepare/install
    tmt test create /install                ---> /install
    tmt test create .                       ---> /tests/prepare
    tmt test create ../execute/basic        ---> /tests/execute/basic
    tmt test create /tests/prepare/install  ---> /tests/prepare/install

In other words, handle relative paths relatively and absolute paths from the `fmf` tree root.
